frontend/package-lock.json

```diff
@@ -1,11750 +0,0 @@
-{
-  "name": "client",
```

Comment:
Do not add changes from package-lock files into commits.
TODO: remove all package-lock files from remote repo

Reply:
@Weizhau @IvanPletinskiy  Check gitignore for those files, especially local IDE settings